### PR TITLE
✨ Use default status visibility for check-ins

### PR DESCRIPTION
**Pull request by DRSchlaubi**
Depending on https://github.com/Traewelling/traewelling/pull/1196

### CODE REVIEW

File: app/src/main/java/de/hbch/traewelling/api/models/user/User.kt [hunk 1]
```diff
@@ -17,5 +17,6 @@ data class User(
     @SerializedName("privateProfile") val privateProfile: Boolean,
     @SerializedName("home") var home: Station?,
     @SerializedName("prevent_index") val preventIndex: Boolean,
-    @SerializedName("language") val language: String
+    @SerializedName("language") val language: String,
+    @SerializedName("default_status_visibility") val defaultStatusVisibility: StatusVisibility
```
Comment (jheubuch) [inline app/src/main/java/de/hbch/traewelling/api/models/user/User.kt:21]:
```suggestion
    @SerializedName("defaultStatusVisibility") val defaultStatusVisibility: StatusVisibility
```

You changed this name in the other PR :)

**Review by jheubuch — CHANGES_REQUESTED** [review]

**Comment by DRSchlaubi** [comment]
forgot to push

**Review by jheubuch — APPROVED** [review]

**Comment by jheubuch** [comment]
Pls do not merge until a new Träwelling release is created

**Comment by DRSchlaubi** [comment]
Change has been deployed

**Pull request merged**